account/list_servers_rpc.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package account;
+
+option go_package = "./;grpc_account";
+
+// * Basically just CreateServerRequest?
```

Comment:
What does this comment mean?

Reply:
If there's this much overlap then maybe we should break this out into it's own type that can be reused? There's LOTS of overlap in these methods tbh, not just the account server ones